src/config.js

```diff
@@ -145,11 +145,11 @@ export default class Config {
 
   nonInteractive: boolean;
 
-  workspacesExperimental: boolean;
+  workspacesEnabled: boolean;
 
   //
   cwd: string;
-  worktreeFolder: ?string;
+  workspaceFolder: ?string;
```

Comment:
I just wanted to not introduce a new term worktree because it does not imply that it is only relevant in workspaces.
workspaceRootFolder sounds quite clear.